packages/block-library/src/index.native.js

```diff
@@ -130,20 +134,52 @@ const registerBlock = ( block ) => {
 	} );
 };
 
+/**
+ * Function to register a block variations e.g. social icons different types.
+ *
+ * @param {Object} block The block which variations will be registered.
+ *
+ */
+const registerBlockVariations = ( block ) => {
+	const { metadata, settings, name } = block;
+
+	settings.variations
+		.sort( ( a, b ) => {
```

Comment:
Maybe instead of implementing own comparator, we could use `lodash.sortBy` with `title` as a key https://lodash.com/docs/#sortBy

```js
_.sortBy(settings.variations, 'title');
```

Reply:
Hmm, makes sense (less code), I will refactor it then.